internal/buffer/save.go

```diff
@@ -106,6 +106,10 @@ func (b *Buffer) saveToFile(filename string, withSudo bool, autoSave bool) error
 		return err
 	}
 
+	b.Path = filename
+	absPath, _ := filepath.Abs(filename)
+	b.AbsPath = absPath
```

Comment:
It was necessary for the following `b.Backup(true)` call, which needed `b.Path` being set, since it works at the same `Buffer`.
Ok, the next two lines aren't necessary, but I decided to keep them at one place in that moment.